src/overlays/actors/ovl_Dm_Char04/z_dm_char04.c

```diff
@@ -28,18 +30,125 @@ const ActorInit Dm_Char04_InitVars = {
     (ActorFunc)DmChar04_Draw,
 };
 
-#endif
+void func_80AABC40(SkelAnime* arg0, AnimationInfo* animation, u16 arg2) {
```

Comment:
```suggestion
void func_80AABC40(SkelAnime* skelAnime, AnimationInfo* animation, u16 index) {
```
You could also name this as `DmChar04_ChangeAnim`